assets/js/components/SaptuneDetails/SaptuneDetails.test.jsx

```diff
@@ -102,4 +105,35 @@ describe('SaptuneDetails', () => {
       staging.solutions_ids
     );
   });
+
+  it('should render all the icons properly', () => {
```

Comment:
I see this test super weak. It should check all the combinations.
I would extract the service logic to `SaptuneService.jsx` component and test them all with a `it.each` kind of test.

Reply:
Good idea, it was challenging to test it, i will move it :D